AmplifyPlugins/Storage/AWSS3StoragePluginFunctionalTests/AWSS3StoragePluginTestBase.swift

```diff
@@ -9,14 +9,14 @@ import XCTest
 import AWSMobileClient
 @testable import Amplify
 import AWSS3StoragePlugin
+import AmplifyPlugins
```

Comment:
had troubles importing AWSAuthPlugin, will take a look at this again

Reply:
i believe this is correct since we are importing AmplifyPlugins pod, even if we are just specifying one of the sub pods